docs/Developers/Development_Setup.rst

```diff
@@ -78,8 +87,8 @@ https://github.com/coala/coala-bears
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip using
+-We recommend first installing the latest development snapshot of coala's
+ master branch from and all of its dependencies with pip using
```

Comment:
@NiklasMM  If you have rights to my commits editing please edit them , I just only try to make it look the same way , I have no idea how others like it .